lib/shopify_api/bulk/query.ex

```diff
@@ -3,6 +3,7 @@ defmodule ShopifyAPI.Bulk.Query do
   alias ShopifyAPI.Bulk.Cancel
 
   @type status_response :: map()
+  @stream_http_timeout 5_000
```

Comment:
Is this a reasonable default for the underlying streaming mechanism? Should it be configurable, given Shopify's renowned response times?

Reply:
This would be for the fetching of the jsonl results file... _should_ be fast 5seconds might even be too long in my books